tests/homework/homework.spec.ts

```diff
@@ -0,0 +1,310 @@
+import { test, expect } from '@playwright/test';
+import { homework } from './homework.page';
+
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('https://travel.agileway.net/login');
+  const landingPage = new homework(page);
+});
+
+
+test.only('Login to the webpage and order flight withh return', async ({page})=>{
```

Comment:
should remove .only()- if merged with this keyword only this single test would run